tsl/test/expected/transparent_decompression-12.out

```diff
@@ -8592,3 +8592,95 @@ SELECT count(*) from motion_table;
 (1 row)
 
 --END of test for page settings
+-- github issue 5585
+create table kon (
+    time timestamptz not null,
+    a varchar(255) not null,
+    b int,
+    c int
+);
+SELECT create_hypertable('kon', 'time');
+WARNING:  column type "character varying" used for "a" does not follow best practices
```

Comment:
What do you think about putting this into a separate file? This one's output is already 9k lines, and it's versioned, it's becoming unwieldy.

Reply:
sure, I will create a new test file for this, is `transparent_decompression_idx_join` a good name? and should this test be versioned as well? I think it would be good to make it a versioned test since `transparent_decompression` is too

Reply:
Yep, that one is smaller. About versions, I try to avoid them when possible, because they are inconvenient to work with.